c7n/version.py

```diff
@@ -11,5 +11,6 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
really shouldn't be here either, this file is a constant variable def, we may gen in the future, we can make it explicit on u"" but i'd rather not carry imports here.